backend/app/core/middleware.py

```
@@ -69,68 +68,22 @@ async def log_system_actions(request: Request, call_next):
-                        extra={"request_id": request_id}
-                    )
-                except Exception as e:
-                    logger.error(
-                        f"Failed to log system action: {str(e)}",
-                        extra={"request_id": request_id}
-                    )
-                    await session.rollback()
+        try:
+            response = await call_next(request)
+        except Exception as e:
+            logger.error(f"Error during call_next: {str(e)}", extra={"request_id": request_id})
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)